prettyprinter/prettyprinter.cabal

```diff
@@ -25,6 +25,10 @@ source-repository head
 library
     exposed-modules:
           Prettyprinter
+        , Prettyprinter.Compat.Text
+        , Prettyprinter.Compat.Text.IO
+        , Prettyprinter.Compat.Text.Lazy
+        , Prettyprinter.Compat.Text.Lazy.Builder
```

Comment:
That's a lot of additional modules for what I had hoped to remain a rather small and hidden corner of the API.

If we add these, I think they should be hidden with ` {-# OPTIONS_HADDOCK hide #-}`